tools/interactive/interactivetool_anylabeling.xml

```diff
@@ -0,0 +1,71 @@
+<tool id="interactive_tool_anylabeling" tool_type="interactive" name="AnyLabeling Interactive" version="@VERSION@" profile="24.2">
+    <description>Effortless data labeling with AI support from YOLO and Segment Anything! AnyLabeling = LabelImg + Labelme + Improved UI + Auto-labeling</description>
+    <requirements>
+	    <container type="docker">quay.io/sunyi000/docker-yololabel:v0.4.16</container>
```

Comment:
Do you want to do the same than last time? The github repo.

Reply:
yes I can do the same as last time.

Reply:
https://github.com/usegalaxy-eu/docker-anylabeling here it is.

Reply:
Thanks @bgruening ,I pushed the files, added the .github/workflows ymls and made a release, but the repo in quay.io still does not exist..is there anything else i need to do to make it appear?

Reply:
I think the CI yml files needs to be in a .github/workflows/ folder

Reply:
https://quay.io/repository/bgruening/anylabeling?tab=settings

Quay.io makes reos hidden by default. I changed it to public. Thanks @sunyi000